website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md

```diff
@@ -208,6 +208,67 @@ the integration between Okta and dbt Cloud.
 3. After creating the Okta application, follow the instructions in the [dbt Cloud Setup](#dbt-cloud-setup)
 section to complete the integration.
 
+## Google integration
+
+_Use this section if you are configuring Google as your identity provider_.
+
+### Configure the Google application
+
+Sign into your **Google Admin Console** via an account with super administrator privileges.
```

Comment:
@mirnawong1 can we add steps for this procedure?

Reply:
done @runleonarun and updated any UI elements that were mentioned too!